### DISCUSSION TITLE
How do I change a shortcut for strikethrough?

### DISCUSSION THREAD

**Discussion by qurle** [Milkdown/milkdown#1538] [Q&A] [answered]
Hi, I create an instance like this
```
const editor = await Editor.make()
  .use(commonmark)
  .use(gfm)
  .use(...)
  .create()
```

There's strikethrough mark I really like and it has own keymap: 
> `Mod-Alt-x` - Toggle the strikethrough mark.

Can I change it to `Mod-Shift-x`? Where do I config that?

I'm kinda dummy, so I'll be glad for the fullest possible code examples. Thanks!

**Comment by shenzhongkang** [comment, marked as answer]
@qurle add this in your `editor.config(ctx => { /* add code here */ })`
```js
 ctx.set(strikethroughKeymap.key, {
    ToggleStrikethrough: ['Mod-Shift-x']
 })
```

**Reply by qurle** [reply]
Thanks!
